lib/check_params.h

```diff
@@ -573,6 +573,11 @@ void printQudaInvertParam(QudaInvertParam *param) {
   }
 #endif
 
+#ifdef INIT_PARAM
+  P(madwf_param_infile[0], '\0');
```

Comment:
Out of curiosity, is this necessary? Or shouldn't the default initialization for a `std::string` always be the empty string?